test/unit/mdc-textfield/mdc-text-field.test.js

```diff
@@ -86,6 +81,30 @@ test('#constructor when given a `mdc-text-field--box` element, initializes a def
   assert.instanceOf(component.ripple, MDCRipple);
 });
 
+const getHelperTextElement = () => bel`<p id="helper-text">helper text</p>`;
+
+test('#constructor instantiates a helper text on the element with id specified in the input aria-controls' +
```

Comment:
We probably don't need this test...I think the following one is more appropriate

Which does beg the question, do we even need the `helperTextFactory` argument at all? Or the `FakeHelperText`?

Reply:
Per offline discussion, removed FakeHelperText and helperTextFactory since we don't need it. The only reason to have a factory argument was to be able to mock methods for testing, which isn't necessary for the helper text.